substrate/client/db/src/utils.rs

```diff
@@ -205,11 +215,11 @@ fn open_database_at<Block: BlockT>(
 		},
 		DatabaseSource::Auto { paritydb_path, rocksdb_path, cache_size } => {
 			// check if rocksdb exists first, if not, open paritydb
-			match open_kvdb_rocksdb::<Block>(rocksdb_path, db_type, false, *cache_size, limit_size)
+			match open_kvdb_rocksdb::<Block>(rocksdb_path, db_type, false, false, *cache_size, limit_size)
```

Comment:
Not sure if forcing `false` is the correct thing to do here - `recreate_onstart` is only set to true if the user explicitly sets it. 
But if they set it and it has no effect (because they don't specify the db) it would be quite surprising.

Reply:
Ok makes sense thanks.